notebook/services/api/api.yaml

```diff
@@ -624,6 +624,18 @@ definitions:
       name:
         type: string
         description: kernel spec name
+      last_activity:
+        type: string
+        description: |
+          ISO 8601 timestamp for the last-seen activity on this kernel.
```

Comment:
They are explicitly timezone-aware, and UTC as an implementation detail. I'll add that detail, though.